src/components/Nav/Mobile/index.tsx

```diff
@@ -14,55 +14,50 @@ import MenuBody from "./MenuBody"
 import MenuFooter from "./MenuFooter"
 import MenuHeader from "./MenuHeader"
 
+import useDisclosure from "@/hooks/useDisclosure"
```

Comment:
(Only if we switch to a named export)

```suggestion
import { useDisclosure } from "@/hooks/useDisclosure"
```